src/options.ts

```diff
@@ -243,6 +244,11 @@ export interface IProps<ListItem = any> {
    */
   FloatingComponent?: ReactNode;
 
+  /**
+   * A reference to a view which provides scroll behaviour
```

Comment:
```suggestion
   * A reference to the view that provides the scroll behavior.
```

Let's use "behavior" since we've been writing it this way since the beginning.

You can change in the doc section as well 👍